python/ray/rllib/ppo/ppo.py

```diff
@@ -281,7 +187,7 @@ def _save(self, checkpoint_dir):
         extra_data = [
             self.local_evaluator.save(),
             self.global_step,
-            self.kl_coeff,
+            self.kl_coeff,   # TODO(rliaw): move this to local evaluator
```

Comment:
can you find where kl_coeff is being handled? It should be updated, but not sure where the final is kept... It may be either in PPO.py, local_evaluator, or nowhere.

Reply:
ppo.py L143: "info["kl_coefficient"] = self.local_evaluator.update_kl(info['kl'])"
should I change it to "info["kl"] = self.local_evaluator.update_kl(info['kl'])" instead?

Reply:
This is a different issue right?

Reply:
Let's remove `kl_coeff` in this class